src/Validation/Validation.php

```diff
@@ -521,9 +521,11 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * To check if a value is set specifically to false or True see `inList()`.
```

Comment:
And how would you communicate that into boolean docblock? Also a nice boolean validation would work for strict or relaxed inputs.

Do you think I should close this and fix boolean() to take an argument to validate against false/'0',0 XOR true/'1'/1 etc?